sklearn/datasets/base.py

```diff
@@ -263,6 +269,9 @@ def load_iris():
         'target_names', the meaning of the labels, 'feature_names', the
         meaning of the features, and 'DESCR', the
         full description of the dataset.
+    
+    (data,target) : tuple if return_X_y == true
+
```

Comment:
Thanks alot. @jnothman did point this fact in the beginning but I failed to understood.
Fixed.